### PR TITLE
Add ability to support return chunk size for splitting purpose

### PR DESCRIPTION
**Pull request by JohnLeSked**

### CODE REVIEW

File: papaparse.js [hunk 1]
```diff
@@ -530,7 +535,11 @@ License: MIT
 			if (IS_PAPA_WORKER)
 			{
 				global.postMessage({
-					results: results,
+					results: Object.assign(results, {
```
Comment (cma-skedulo) [inline papaparse.js:538]: @JohnLeSked Object.assign will mutate the first argument
so this whole block is similar to
```
result.meta.currentByteCursor = this._currentByteCursor
```

File: papaparse.js [hunk 2]
```diff
@@ -522,6 +526,7 @@ License: MIT
 				this._baseIndex = lastIndex;
 			}
 
+			 this._currentByteCursor += this._partialLine ? this.calculateTotalByteByString(aggregate) - this.calculateTotalByteByString(this._partialLine) : this.calculateTotalByteByString(aggregate)
```
Comment (cma-skedulo) [inline papaparse.js:529]: out of curiousity, why we don't handle this inside Parser.parse? and return currentByteCursor inside results.meta

File: papaparse.js (hunk 1 above)
Comment (JohnLeSked) [reply papaparse.js:538]: Yeah, i put the wrong target, source order

File: papaparse.js (hunk 2 above)
Comment (JohnLeSked) [reply papaparse.js:529]: This is not an optimal solution indeed, will make changes later today to put logic inside parse() function.

**Pull request merged**